src/worker/coverage.ts

```diff
@@ -66,6 +66,7 @@ export class VitestCoverage {
     this.ctx.logger.log('Running coverage with configuration:', this.config)
 
     if (!this._provider) {
+      vitest.config.coverage.enabled = true
```

Comment:
`enabled` is configured by `this.enabled`, it should be a getter referencing `this._enabled`